irspack/optimizers/_optimizers.py

```diff
@@ -92,6 +93,25 @@ def tune_range_given_memory_budget(
         return []
 
 
+class EDLAEOptimizer(BaseOptimizer):
+    default_tune_range: List[Suggestion] = [
+        LogUniformSuggestion("reg", 1, 1e4),
+        UniformSuggestion("dropout_p", 0.05, 0.40),
```

Comment:
Is there any rationale for the upper and lower bound of the `dropout_p` parameters?
I think setting larger range `UniformSuggestion("dropout_p", 0.0, 0.99)` will not particularly increase 
the cost of parameter tuning.

Reply:
I used the range 0.05-0.40 because EDLAE author, Harald Steck used relatively small value
(0.10 for MSD data, 0.15 for NetFlix data, 0.33 for ML-20M data) in his [notebook](https://github.com/hasteck/EDLAE_NeurIPS2020/blob/main/EDLAE_NeurIPS2020.ipynb),
which are determined based on parameter tuning, I think.
But your suggestion is quite right. I changed tune range of dropout_p to 0.0-0.99.
I checked modified EDLAE optimizer still generated better nDCG@20 than DenseSLIM (for ML-1M data).
Would you please check it again?